SPECS/envoy/envoy.spec

```diff
@@ -0,0 +1,440 @@
+# - https://github.com/grpc/grpc/archive/d8f4928fa779f6005a7fe55a176bdb373b0f910f.tar.gz
+# - https://github.com/jbeder/yaml-cpp/archive/yaml-cpp-0.6.3.tar.gz
+# - https://github.com/libevent/libevent/archive/0d7d85c2083f7a4c9efe01c061486f332b576d28.tar.gz
+# - https://github.com/lightstep/lightstep-tracer-cpp/archive/3efe2372ee3d7c2138d6b26e542d757494a7938d.tar.gz
+# - https://github.com/mirror/tclap/archive/tclap-1-2-1-release-final.tar.gz
+# - https://github.com/moonjit/moonjit/archive/2.2.0.tar.gz
+# - https://github.com/msgpack/msgpack-c/releases/download/cpp-3.2.1/msgpack-3.2.1.tar.gz
+# - https://github.com/nodejs/http-parser/archive/v2.9.3.tar.gz
+# - https://github.com/opentracing/opentracing-cpp/archive/v1.5.1.tar.gz
+# - https://github.com/openzipkin/zipkin-api/archive/0.2.2.tar.gz
+# - https://github.com/pallets/jinja/archive/2.10.3.tar.gz
+# - https://github.com/pallets/markupsafe/archive/1.1.1.tar.gz
+# - https://github.com/prometheus/client_model/archive/99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c.tar.gz
+# - https://github.com/protocolbuffers/protobuf-go/archive/v1.22.0.zip
+# - https://github.com/protocolbuffers/protobuf/releases/download/v3.10.1/protobuf-all-3.10.1.tar.gz
+# - https://github.com/protocolbuffers/upb/archive/8a3ae1ef3e3e3f26b45dec735c5776737fc7247f.tar.gz
+# - https://mirror.bazel.build/github.com/bazelbuild/platforms/archive/9ded0f9c3144258dad27ad84628845bcd7ca6fe6.zip
+# - https://mirror.bazel.build/github.com/bazelbuild/rules_java/archive/7cf3cefd652008d0a64a419c34c13bdca6c8f178.zip
+# - https://mirror.bazel.build/github.com/golang/protobuf/archive/v1.4.1.zip
+# - https://mirror.bazel.build/github.com/golang/tools/archive/2bc93b1c0c88b2406b967fcd19a623d1ff9ea0cd.zip
+# - https://mirror.bazel.build/github.com/protocolbuffers/protobuf-go/archive/v1.22.0.zip
+Source1:        %{name}-vendor-%{version}.tar.gz
+# END obs-service-bazel_repositories
+Source100:      %{name}-rpmlintrc
+Patch0:         0001-build-Use-Go-from-host.patch
+Patch1:         0002-build-update-several-go-dependencies-11581.patch
+Patch2:         0003-build-Add-explicit-requirement-on-rules_cc.patch
+# AUTOGENERATED BY obs-service-bazel_repositories
+Provides:       bundled(abseil-cpp) = 06f0e767d13d4d68071c4fc51e25724e0fc8bc74
+Provides:       bundled(apple_support) = 0.7.2
+Provides:       bundled(bazel-gazelle) = 0.19.1
+Provides:       bundled(bazel-skylib) = 0.9.0
+Provides:       bundled(bazel-toolchains) = 2.2.0
+Provides:       bundled(c-ares) = d7e070e7283f822b1d2787903cce3615536c5610
+Provides:       bundled(cel-cpp) = 80e1cca533190d537a780ad007e8db64164c582e
+Provides:       bundled(client_model) = 99fa1f4be8e564e8a6b613da7fa6f46c9edafc6c
+Provides:       bundled(dd-opentracing-cpp) = 1.1.3
+Provides:       bundled(envoy-build-tools) = 84ca08de00eedd0ba08e7d5551108d6f03f5d362
+Provides:       bundled(fmt) = 6.0.0
+Provides:       bundled(googleapis) = 82944da21578a53b74e547774cf62ed31a05b841
```

Comment:
Please also add instructions on how to generate these `Provides: bundled(X)`. I'm guessing it has something to do with that mysterious `obs-service-bazel_repositories` executable (I think?) mentioned above, but we could use more detailed, step-by-step instructions.